avm/res/network/virtual-network/subnet/main.bicep

```diff
@@ -3,7 +3,7 @@ metadata description = 'This module deploys a Virtual Network Subnet.'
 metadata owner = 'Azure/module-maintainers'
 
 @description('Optional. The Name of the subnet resource.')
-param name string
+param name string = ''
```

Comment:
```suggestion
param name string
```

Reply:
@blinq-erik please review the changes, and update the title of the PR please. Thanks!